packages/proxy/src/proxy.ts

```diff
@@ -65,6 +71,12 @@ export interface CacheKeyOptions {
   excludeOrgName?: boolean;
 }
 
+export interface SpanLogging {
+  setName: (name: string) => void;
+  log: (args: ExperimentLogPartialArgs) => void;
+  end: () => void;
+}
```

Comment:
do we need to explicitly flush the logger here, or is it guaranteed that the `beforeExit` event will fire in all environments that we run this?

Reply:
I think it's ok to make it the responsibility of the caller (which is currently our own tightly controlled server, where `asyncFlush` is false by default anyway).